test/index.js

```diff
@@ -161,7 +162,7 @@ describe('Token', function () {
     var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', { expiresIn: -10 }) } };
 
     server.inject(request, function (res) {
-      expect(res.result.message).to.equal('Expired token received for JSON Web Token validation');
+      expect(res.result.message).to.equal('jwt expired');
```

Comment:
this message is too short. think that will be descriptive enough? same for next one